changelog/1.get_merged_prs.py

```diff
@@ -19,6 +19,8 @@ def parse_isoformat(string):
 
 REPOSITORY = 'astropy/astropy'
 
+YESTERDAY = datetime(2016, 12, 13, 0, 0, 0)
```

Comment:
Probably this should be `datetime.datetime.now()-datetime.timedelta(1)`.  Will just make that directly in this branch.